packages/grafana-ui/src/types/panel.ts

```diff
@@ -5,11 +5,16 @@ import { ScopedVars } from './datasource';
 
 export type InterpolateFunction = (value: string, scopedVars?: ScopedVars, format?: string | Function) => string;
 
+export type WithSeriesOptions<T extends {}, S> = T & {
```

Comment:
I am not sure this interface will work, not every options on a panel will have a series override.